aspnetcore/blazor/get-started.md

```diff
@@ -16,6 +16,111 @@ By [Daniel Roth](https://github.com/danroth27) and [Luke Latham](https://github.
+
+   3\. For a Blazor WebAssembly experience, execute the following command in a command shell:
+
+      ```dotnetcli
+      dotnet new blazorwasm -o WebApplication1
+      ```
+
+      For a Blazor Server experience, execute the following command in a command shell:
+
+      ```dotnetcli
+      dotnet new blazorserver -o WebApplication1
+      ```
+
+      For information on the two Blazor hosting models, *Blazor Server* and *Blazor WebAssembly*, see <xref:blazor/hosting-models>.
+
+   4\. Open the *WebApplication1* folder in Visual Studio Code.
+
+   5\. For a Blazor Server project, the IDE requests that you add assets to build and debug the project. Select **Yes**.
+
+   6\. If using a Blazor Server app, run the app using the Visual Studio Code debugger. If using a Blazor WebAssembly app, execute `dotnet run` from the app's project folder.
+
+   7\. In a browser, navigate to `https://localhost:5001`.
+
+   <!--
+
+   # [Visual Studio for Mac](#tab/visual-studio-mac)
+
+   1\. Install [Visual Studio for Mac](https://visualstudio.microsoft.com/vs/mac/). Switch the [Update channel to Preview](/visualstudio/mac/install-preview).
+
+   2\. Select **File** > **New Solution** or **New Project**.
+
+   3\. In the sidebar, select **.NET Core** > **App**.
+
+   4\. For a Blazor Server experience, select the **Blazor Server App** template. For a Blazor WebAssembly experience, select the **Blazor WebAssembly App** template. Select **Next**. For information on the two Blazor hosting models, *Blazor Server* and *Blazor WebAssembly*, see <xref:blazor/hosting-models>.
+
+   5\. The **Target Framework** defaults to **.NET Core 3.0**. Select **Next**.
+
+   6\. In the **Project Name** field, enter `WebApplication1`. Select **Create**.
+
+   7\. Select **Run** > **Run Without Debugging** to run the app *without the debugger*. Running with the debugger isn't supported at this time.
+
+   -->
+
+   # [.NET Core CLI](#tab/netcore-cli/)
+
+   For a Blazor WebAssembly experience, execute the following commands in a command shell:
+
+   ```dotnetcli
+   dotnet new blazorwasm -o WebApplication1
+   cd WebApplication1
+   dotnet run
+   ```
+
+   For a Blazor Server experience, execute the following commands in a command shell:
+
+   ```dotnetcli
+   dotnet new blazorserver -o WebApplication1
+   cd WebApplication1
+   dotnet run
+   ```
+
+   For information on the two Blazor hosting models, *Blazor Server* and *Blazor WebAssembly*, see <xref:blazor/hosting-models>.
+
+   In a browser, navigate to `https://localhost:5001`.
+
+   ---
+
+::: moniker-end
+
+::: moniker range="< aspnetcore-3.1"
+
 1. Install the latest [.NET Core 3.0 SDK](https://dotnet.microsoft.com/download/dotnet-core/3.0) release.
 
 1. Install the [Blazor WebAssembly](xref:blazor/hosting-models#blazor-webassembly) template by running the following command in a command shell:
```

Comment:
For the 3.0, we still want to give updated instructions on how to install Blazor WebAssembly using .NET Core 3.1. This seems like the most natural spot to do that. We should update this step to tell folks to 1. Install .NET Core 3.1 Preview, and 2. run the command to install the latest version of the Blazor WebAssembly templates (3.1.0-preview1.blah). Something like this:

```
1. Optionally install the [Blazor WebAssembly](xref:blazor/hosting-models#blazor-webassembly) template by installing the [.NET Core 3.1 Preview SDK](https://dotnet.microsoft.com/download/dotnet-core/3.1) and then running the following command in a command shell:
```

In the VS section of the 3.0 docs we also need to tell folks using Blazor WebAssembly to install VS 2019 16.4.